_data/toc.yaml

```diff
@@ -1182,8 +1182,32 @@ manuals:
               title: Edge releases
             - path: /desktop/windows/release-notes/archive/
               title: Older versions
-    - path: /desktop/linux/
-      title: Linux (Beta)
+    - sectiontitle: Linux
+      section:
```

Comment:
Perhaps we should move the "user manual" item to the top of the list; it's currently the 4th item, but it's the first page that opens at `/desktop/linux/`. As it's the "opening page", it probably makes sense to put it at the top.

<img width="255" alt="Screenshot 2022-05-05 at 23 51 52" src="https://user-images.githubusercontent.com/1804568/167031590-d86923e6-a2e8-48d6-bab5-188bb330bfcc.png">

Reply:
Set this as the first item